scripts/beegfspkgs-713.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
How much differences differences exists in the scripts between versions ?
Can't you just use the same script and have the version number as a parameter with a default value to keep existing config file untouched. This will make easier the update for future releases

Reply:
yes, good point. I am hoping this will be a temporary fix and that ba ug fix will be released from thinkparQ and we can eventually remove this special version.

Reply:
I removed beegfspkgs-713.sh and added changed to beegfspkgs.sh (so beegfs examples do not need to be modified).